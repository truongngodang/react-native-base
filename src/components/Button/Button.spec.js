/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import { Button } from '..';

describe('Button Component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
