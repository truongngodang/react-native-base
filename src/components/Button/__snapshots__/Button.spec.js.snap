// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders as expected 1`] = `ShallowWrapper {}`;
